src/resources/postcss/components/_messages.pcss

```diff
@@ -0,0 +1,53 @@
+.tribe-events {
+	/* -----------------------------------------------------------------------------
+	 *
+	 * Messages
+	 *
+	 * Example:
+	 * <div class="tribe-events-c-messages">
+	 *   <ul class="tribe-events-c-messages__list">
+	 *     <li class="tribe-events-c-messages__list-message">There were no results found</li>
+	 *   </ul>
+	 * </div>
+	 *
+	 * ----------------------------------------------------------------------------- */
+
+	.tribe-events-c-messages {
+		align-items: center;
+		background-color: var(--color-background-messages);
+		border-radius: var(--border-radius-default);
+		display: flex;
+		margin: 0 0 var(--spacer-3) auto;
+		order: -1;
```

Comment:
Let's move the message HTML above the events bar so we don't have to set order twice, once on mobile and again on desktop. It's a good idea to try to mess with order as little as possible.